ti/callgraph/upload.go

```diff
@@ -26,34 +26,19 @@ const (
 )
 
 // Upload method uploads the callgraph.
-func Upload(ctx context.Context, stepID string, timeMs int64, log *logrus.Logger, start time.Time) error {
-	if instrumentation.IsManualExecution() {
+func Upload(ctx context.Context, stepID string, timeMs int64, log *logrus.Logger, start time.Time, cfg *tiCfg.Cfg) error {
+	if instrumentation.IsManualExecution(cfg) {
 		log.Infoln("Skipping call graph collection since it is a manual run")
 		return nil
 	}
 
-	cfg := pipeline.GetState().GetTIConfig()
-	if cfg == nil || cfg.URL == "" {
-		return fmt.Errorf("TI config is not provided in setup")
-	}
-
-	source := cfg.SourceBranch
-	if source == "" {
-		return fmt.Errorf("source branch is not set")
-	}
-	target := cfg.TargetBranch
-	if target == "" {
-		return fmt.Errorf("target branch is not set")
-	}
-
 	encCg, err := encodeCg(fmt.Sprintf(cgDir, pipeline.SharedVolPath), log)
```

Comment:
Change this sharedVolPath to use ti config one

Reply:
Good catch. Done